lib/css_parser/rule_set.rb

```diff
@@ -22,6 +22,39 @@ class RuleSet
       ['border-width', %w[border-top-width border-right-width border-bottom-width border-left-width]]
     ].freeze
 
+    class DeclarationValue
+      attr_reader :value
+      attr_accessor :important, :order
+
+      def initialize(value, important: nil, order: 0)
+        self.value = value
+        @important = !value.match(CssParser::IMPORTANT_IN_PROPERTY_RX).nil?
+        @important = important unless important.nil?
+        @order = order
+      end
```

Comment:
- this seems to do a lot of work for cases were we don't need it
- important can just be true/false no need to dead with nils ?